### ISSUE TITLE
Error EOF

### ISSUE THREAD

**Issue by starboyvarun** [tensorflow/tensorflow#51671]
![1](https://user-images.githubusercontent.com/26819449/130785201-768b1e8d-6897-40f2-9501-149ea3eca737.JPG)

![2](https://user-images.githubusercontent.com/26819449/130785213-978c460b-247b-4f41-96b8-447ec1c6e15c.JPG)

Hello,
Actually, I am trying to run a library from the Github repo.
All modules of the application are running Except this one. I haven't found any syntax error.
and code was not altered.
Can you tell me what's wrong?
Thankyou.

**Comment by sushreebarsa** [comment]
@starboyvarun In order to reproduce the issue reported here, could you please provide the complete code and the dataset , tensorflow version you are using. Thanks!

**Comment by starboyvarun** [comment]
I have already shared the complete code.
tf version =2.2
this is the implementation of tutorials you told me to do of janggu.

**Comment by starboyvarun** [comment]
can you please assign this to someone else as you can't already solve this?
It will safe both of our time.
Thank you.

**Comment by sushreebarsa** [comment]
@sanatmpa1 Was able to reproduce this issue on colab using TF v2.2 & 2.6 please find the gist [here](https://colab.research.google.com/gist/sushreebarsa/87b5a39e99b67b44adf41d218f79c29a/untitled408.ipynb#scrollTo=-ve9mZ-aQgTN) for reference. The original issue ticket is #51651 . Thank you!

**Comment by sanatmpa1** [comment]
@starboyvarun,

The straight forward issue that I see in the code is missing parantheses `)` in the `model.fit` line.

Replace the `model.fit(DNA, ReduceDim(LABELS, epochs=100)` with `model.fit(DNA, ReduceDim(LABELS, epochs=100))` and let me know if it works. Thanks!
